lib/twinklyhaha_web/controllers/oc2_controller.ex

```diff
@@ -7,383 +7,27 @@ defmodule TwinklyhahaWeb.OC2Controller do
   @off "off"
 
   def command(conn, params) do
-    Logger.debug("oc2_controller command #{inspect(params)}")
-    ## check top level components of command json
-    tops = Map.keys(params)
-
-    cond do
-      ## is action missing?
-      "action" not in tops ->
-        ## :unprocessable_entity - 422
-        Logger.debug("command no action #{inspect(params)}")
-        Logger.debug("command no action #{inspect(conn)}")
-        send_resp(conn, :unprocessable_entity, "Oops! no action?")
-
-      ## is target missing?
-      "target" not in tops ->
-        Logger.debug("command no target #{inspect(params)}")
+    command =
+      params
+      |> Jason.encode!()
```

Comment:
What happens if this throws an error? What response does the user get? I'm thinking if there is need to catch it or let it blow

Reply:
If this throws an error the command is not created, e.g maybe an empty param is sent, this is the struct we receive back 
```%Openc2.Oc2.Command{
  error?: true,
  error_msg: "no action in command",
  cmd: nil,
  action: nil,
  target: nil,
  target_specifier: nil,
  actuator: nil,
  actuator_specifier: nil,
  cmd_id: nil,
  args: nil,
  response: nil
}